File: Clojure/pe-001.clj
(defn sum-multiples-below [k n]
	"sum of all the multiples of k below n"
	(->> (range k n k)
	     (reduce +)))

(println (- (+ (sum-multiples-below 3 1000) 
	           (sum-multiples-below 5 1000)) 
	        (sum-multiples-below 15 1000)))

File: Python/pe_001.py
# project euler-001

def sum(a,n):
    b=int((n-1)/a)
    s=a*b*(b+1)//2
    return int(s)
    
t=int(input())

for i in range(0,t):
    n=int(input())
    print(int(sum(3,n)+sum(5,n)-sum(15,n)))
